src/translations/ru.json

```
@@ -0,0 +1,39 @@
+{
+  "log_in": "Войти",
+  "log_out": "Выйти",
+  "logged_in_as": "Вы вошли как",
+  "logged_in": "Вы вышли",
+  "logging_in": "Вхожу",
+  "logging_out": "Выхожу",
+  "sign_up": "Зарегистрироваться",
+  "signing_up": "Идет регистрация",
```

Here, the indefinite form is used correctly the way I would suggest to use it in other places. But I think the word "идёт" (going) has limited usage in this context and cannot provide consistency in all the occurrences here. Thus, "выполняется" (being executed)  seems to be a more convenient choice with the same meaning.

```suggestion
  "signing_up": "Выполняется регистрация",
```